packages/solid/components/Artwork/Artwork.stories.tsx

```diff
@@ -23,11 +23,31 @@ import theme from 'theme';
 const meta: Meta<typeof Artwork> = {
   title: 'Components/Artwork',
   component: Artwork,
-  tags: ['autodocs'],
   argTypes: {
+    src: {
+      description: 'source URI of the image to be displayed',
+      control: { type: 'text' }
+    },
+    fallbackSrc: {
+      description: 'URI of fallback image to be shown if the src request fails',
+      control: { type: 'text' }
+    },
     fillColor: {
       description: 'solid color background, displayed if src undefined or invalid',
-      control: 'color'
+      control: { type: 'color' }
+    },
+    gradient: {
+      description: 'enable to display a gradient overlay, color can be set with `gradientColor`',
+      control: { type: 'boolean' },
+      default: 'false'
+    },
+    gradientColor: {
+      description: 'Color used in gradient ramp when `gradient="true"`',
+      control: { type: 'color' }
+    },
+    srcCallback: {
+      description:
+        'optional callback function that can be used to generate custom strings to request an image. The callback will be passed an object containing the following parameters: `aspectRatio`, `src`, `w`, `h`. Be default aspect ratio will match the closest value from srcCallbackAspectRatios'
```

Comment:
just checking, do we want to set up controls for this arg or no?

![Screenshot 2024-09-18 at 12 35 15 PM](https://github.com/user-attachments/assets/2df428db-5e90-4351-9d4b-cf550dbbd845)

Reply:
In LUI, there's a boolean to toggle whether one is being applied, probably just so that we had some code publicly available to demonstrate how it works.

@anthony9187 mentioned recipes yesterday. Maybe this is a good use case for those?

Reply:
so my reason for putting this here was to have APIs documented in the args panel so it can be a quick reference even if the actual arg cant be controlled. we could probably remove it in favor of relying on the Docs > API table

outside that, it could be a good example for a Recipe. perhaps even a composed Tile with fallbacks and everything(a Bells and Whistles example, if you will)